### PR TITLE
Rename `x` and `y` to prepare for Chapter 5

### PR DESCRIPTION
**Pull request by pavpanchekha**
Chapter 5 wants the `x` and `y` fields to refer to the position of boxes. This means the text cursor needs to use some other variable names. This PR changes all similar variables earlier in the book to use `row` and `col`. These aren't ideal, but they make sense when introduced (because it is laying out Chinese text in a grid) and they go away eventually (in Chapter 7, which adds inline layout), and we don't use `row` / `column` anywhere else.

### CODE REVIEW

File: book/graphics.md [hunk 1]
```diff
@@ -212,15 +212,21 @@ every letter in the same place, so they all overlap! Let's fix that:
 
 ``` {.python expected=False}
 HSTEP, VSTEP = 13, 18
-x, y = HSTEP, VSTEP
+cx, cy = HSTEP, VSTEP
```
Comment (chrishtr) [inline book/graphics.md:215]: What does cx stand for?

Comment (pavpanchekha) [reply book/graphics.md:215]: They text tries to explain it as "cursor x" / "cursor y".

Comment (chrishtr) [reply book/graphics.md:215]: cursor_x then?

File: book/graphics.md [hunk 2]
```diff
@@ -240,15 +240,21 @@ every letter in the same place, so they all overlap! Let's fix that:
 
 ``` {.python expected=False}
 HSTEP, VSTEP = 13, 18
-x, y = HSTEP, VSTEP
+row, col = VSTEP, HSTEP
```
Comment (chrishtr) [inline book/graphics.md:243]: Row and col seem odd because they refer to tables. How about cursor_x and cursor_y?

**Comment by pavpanchekha** [comment]
I'm not thrilled with `row` and `col`, but `cursor_x` and `cursor_y` introduce a new term ("cursor") and are sort of confusable with `x` and `y`, which in the layout chapter come to mean the overall paragraph's coordinates.

**Comment by chrishtr** [comment]
> I'm not thrilled with `row` and `col`, but `cursor_x` and `cursor_y` introduce a new term ("cursor") and are sort of confusable with `x` and `y`, which in the layout chapter come to mean the overall paragraph's coordinates.

Some more options: char_x and char_y? inline_x, inline_y? offset_x, offset_y?

**Comment by pavpanchekha** [comment]
Merging, we discussed this at the last meeting.

**Pull request merged**